src/security-council-mgmt/governors/SecurityCouncilNomineeElectionGovernor.sol

```diff
@@ -0,0 +1,303 @@
+        IVotesUpgradeable _token,
+        address _owner,
+        uint256 _quorumNumeratorValue,
+        uint256 _votingDelay,
+        uint256 _votingPeriod
+    ) public initializer {
+        __Governor_init("Security Council Nominee Election Governor");
+        __GovernorVotes_init(_token);
+        __SecurityCouncilNomineeElectionGovernorCounting_init();
+        __ArbitrumGovernorVotesQuorumFraction_init(_quorumNumeratorValue);
+        __GovernorSettings_init(_votingDelay, _votingPeriod, 0);
+        _transferOwnership(_owner);
+
+        targetNomineeCount = _targetNomineeCount;
+        firstCohort = _firstCohort;
+        firstNominationStartTime = _firstNominationStartTime;
+        nominationFrequency = _nominationFrequency;
+        foundationBlacklistDuration = _foundationBlacklistDuration;
+        foundation = _foundation;
+        securityCouncilManager = _securityCouncilManager;
+    }
+
+    /// @notice Allows the foundation to call certain functions
+    modifier onlyFoundation {
+        require(msg.sender == foundation, "Only the foundation can call this function");
+        _;
+    }
+
+    /// @notice Allows the owner to make calls from the governor
+    /// @dev    See {L2ArbitrumGovernor-relay}
+    function relay(address target, uint256 value, bytes calldata data)
+        external
+        virtual
+        override
+        onlyOwner
+    {
+        AddressUpgradeable.functionCallWithValue(target, data, value);
+    }
+
+    /// @notice Always reverts.
+    /// @dev    `GovernorUpgradeable` function to create a proposal overridden to just revert. 
+    ///         We only want proposals to be created via `createElection`.
+    function propose(
+        address[] memory,
+        uint256[] memory,
+        bytes[] memory,
+        string memory
+    ) public virtual override returns (uint256) {
+        revert("Proposing is not allowed, call createElection instead");
+    }
+
+    /// @notice Normally "the number of votes required in order for a voter to become a proposer." But in our case it is 0.
+    /// @dev    Since we only want proposals to be created via `createElection`, we set the proposal threshold to 0.
+    ///         `createElection` determines the rules for creating a proposal.
+    function proposalThreshold() public view virtual override(GovernorSettingsUpgradeable, GovernorUpgradeable) returns (uint256) {
+        return 0;
+    }
+
+    /// @notice Creates a new nominee election proposal. 
+    ///         Can be called by anyone every `nominationFrequency` seconds.
+    /// @return proposalIndex The index of the proposal
+    /// @return proposalId The id of the proposal
+    function createElection() external returns (uint256 proposalIndex, uint256 proposalId) {
+        require(block.timestamp >= firstNominationStartTime + nominationFrequency * proposalCount, "Not enough time has passed since the last election");
+
+        proposalIndex = proposalCount;
```

Comment:
yeah that makes sense actually, since there's only one election at a time, we could totally replace `proposalIndex` with `proposalCount - 1` right?

only weird thing i can think of is if for some reason, `_execute` is reverting for the March cohort because of `randomAddToSet` and no action is taken to fill the vacant SC seat to make that not revert before the september election rolls around. At this point, as soon as `proposalCount` is incremented, `execute` is callable with the march `proposalId`, but cohort will be set to september. 

Easy mitigation would be to add a `proposalTimeout` where a proposal becomes invalid some time after the blacklisting deadline. I think getting rid of `proposalIndex`, using `proposalCount - 1` instead, and adding a timeout would be intuitive?